### PR TITLE
Added/Updated Thumbnails

### PR DESCRIPTION
**Pull request by marverix**
screenshot_1549317890

### CODE REVIEW

File: app/src/main/res/drawable/thumb_ubnt.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        android:strokeColor="#00000000" android:strokeWidth="1.33333325"/>
+    <path android:fillAlpha="1" android:fillColor="#39a2e1"
+        android:fillType="nonZero"
+        android:pathData="M7.123,0L7.123,7.113L14.238,7.113L14.238,0ZM28.473,7.113v7.035h7.125L35.598,7.113ZM21.398,14.221v7.039h7.033v-7.039zM35.598,17.74v7.041h7.115L42.713,17.74ZM7.123,19.467L7.123,77.475C7.123,106.32 28.241,128 64.004,128c0,0 -21.291,-11.734 -21.291,-36.063L42.713,40.674L35.598,40.674L35.598,66.906L28.471,66.906L28.471,51.227h-7.107v7.039L14.236,58.266L14.236,19.467ZM28.514,26.598v7.039h7.043v-7.039zM21.398,37.154v7.037L28.437,44.191v-7.037z"
+        android:strokeColor="#00000000" android:strokeWidth="1.33333325"/>
+    <path
+        android:pathData="m68.997,106.605c-9.495,-0.431 -16.8,-2.841 -22.46,-6.327v0c3.833,16.636 18.166,24.806 18.858,25.194v0l4.226,2.33c32.332,-2.159 51.256,-23.032 51.256,-50.325v0,-5.915c-7.685,25.073 -25.57,36.228 -51.879,35.043"
+        android:strokeColor="#00000000" android:strokeWidth="1.33333325">
+        <aapt:attr name="android:fillColor">
+            <gradient android:endX="120.87542" android:endY="99.68164"
```
Comment (flocke) [inline app/src/main/res/drawable/thumb_ubnt.xml]: Gradients can cause a lot of troubles in our experience. Would it be possible to remove it?

Comment (marverix) [reply app/src/main/res/drawable/thumb_ubnt.xml]: You ask me to modify 3rd party logo? I see 2 possibilites: remove this logo or find version without gradient then.

Comment (flocke) [reply app/src/main/res/drawable/thumb_ubnt.xml]: Yeah, that's usually what we try to do (finding a version without gradient). Maybe there is a monochrome version or something?

**Comment by marverix** [comment]
I have updated (force pushed to keep repo clean) vectors. Also I have used Inkscape optimization to minimize size of files
screenshot_1550182624

**Comment by flocke** [comment]
Thank you for changing the Ubiquiti icon, this one looks good as well.
I just noticed that the new Firefox logo also has gradients in it, maybe we should keep the old one for now and look if we can find a version without gradients?

**Comment by marverix** [comment]
Hey, I know about Firefox, but I did research yesterday and I don't really think that there is a lot of what we can do. https://design.firefox.com/photon/visuals/product-identity-assets.html
My only idea is to:
1. Use "Icon as Glyph" and choose some background for it (simmilar like for Ubiquiti).
2. Write to them directly and ask for advice.

**Comment by flocke** [comment]
@marverix Yeah, I will think about it. Maybe for now just remove the new icon from your PR and keep the old one until we figured this out. So we at least can merge the other icons already.

**Comment by marverix** [comment]
Fixed. BTW: Why `.idea` is not ignored??? Any good reason for that?

**Pull request merged**